src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -68,6 +70,10 @@ UserInfoPopup::UserInfoPopup()
                 .assign(&this->ui_.followerCountLabel);
             vbox.emplace<Label>(TEXT_CREATED)
                 .assign(&this->ui_.createdDateLabel);
+            if (getSettings()->showUserIDsInUsercards)
+            {
+                            vbox.emplace<Label>(TEXT_USER_ID).assign(&this->ui_.userIDLabel);
+                        }

```

Comment:
I think it would be cool to reduce the indentation.

Reply:
And this reminds me to run clang-format.